arwen/contexts/output.go

```diff
@@ -196,6 +196,16 @@ func (context *outputContext) Transfer(destination []byte, sender []byte, gasLim
 		return arwen.ErrTransferInsufficientFunds
 	}
 
+	payable, err := context.host.Blockchain().IsPayable(destination)
+	if err != nil {
+		return err
+	}
+
+	hasValue := value.Cmp(big.NewInt(0)) == 1
```

Comment:
It would be nice to have some functions in `helpers.go` such as `bigIntGreaterThan()` - not for now.